shared/common-adapters/modal/index.tsx

```diff
@@ -42,6 +42,7 @@ type Props = {
   onClose?: () => void // desktop non-fullscreen only
   footer?: FooterProps
   fullscreen?: boolean // desktop only. disable the popupdialog / underlay and expand to fit the screen
+  fullWidthScrollview?: boolean // only affects tablet, disables max width for scrollview content when provided
```

Comment:
Let's not add this bool. Let's instead allow storing the scroll content style. Also all this scrollview stuff is going to go away and you'll have to pass your own scrollview anyways

Reply:
yeah i was thinking about that, wasn't sure if modal2 was ready yet though